### ISSUE TITLE
Error from console: I.pluginRunTagRoute is not a function

### ISSUE THREAD

**Issue by theyiwenliu** [tensorflow/tensorboard-plugin-example#22]
After launching greeter plugin, content is empty:

<img width="1673" alt="screen shot 2018-02-28 at 3 47 38 pm" src="https://user-images.githubusercontent.com/10496910/36819606-f8b7c85a-1c9e-11e8-8c7f-540303419a5b.png">

And found error in console:
```
(index):5175 Uncaught TypeError: I.pluginRunTagRoute is not a function
    at HTMLElement._fetchNewData ((index):5175)
    at HTMLElement.reload ((index):5174)
    at HTMLElement.attached ((index):5174)
    at (index):887
    at Object.whenReady ((index):881)
    at HTMLElement.attachedCallback ((index):886)
    at HTMLElement.attachedCallback ((index):924)
    at Object.insertBefore ((index):939)
    at a.insertBefore ((index):946)
    at HTMLTemplateElement._insertInstance ((index):1185)
```

TF: 1.4.1
Bazel: 0.5.4

**Comment by jart** [comment]
That particular thing was removed in https://github.com/tensorflow/tensorboard/pull/494. I'm actually writing a PR now to bump this repo up to the latest state, to fix https://github.com/tensorflow/tensorboard-plugin-example/issues/21. I might be able to fix this alongside.

**Comment by theyiwenliu** [comment]
Fixed by #27
